casper/src/test/resources/updateRegistry/AgreeAuthKey.rho

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Please remove files that are related to multi signature confirmation of update. We can keep these files in a new draft PR for future update.

Reply:
This is only for updating systemContractExample. It is safe to keep.

Reply:
We have three files Agree... Propose... and Update... Are all three used?